commands/note.go

```diff
@@ -0,0 +1,96 @@
+package commands
+
+import (
+	"errors"
+	"fmt"
+
+	pickErrors "github.com/bndw/pick/errors"
+	"github.com/bndw/pick/utils"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+const (
+	defaultNoteAction = "edit"
+)
+
+func init() {
+	cmd := &cobra.Command{
+		Use:   "note [name]",
+		Short: "Create a note",
+		Long:  "The note command is used to create, modify, list and remove secure notes.",
+		Run: func(cmd *cobra.Command, args []string) {
+			runCommand(Note, cmd, args)
+		},
+	}
+	cmd.Flags().Bool("edit", false, "create or modify a note")
+	cmd.Flags().Bool("ls", false, "list all notes")
+	cmd.Flags().Bool("export", false, "export all notes")
```

Comment:
Should we keep `export` separate to the safe export? (i.e. in addition to exporting accounts, `$ pick export` could export notes as well)